apps/web/src/screens/Explore/index.tsx

```diff
@@ -30,8 +30,7 @@ export const ExploreScreen: FC = () => {
 		console.log(extension);
 	};
 
-	const handleAddPress = async (extension: ExtensionDocument) => {
-		console.log('add layout');
+	const handleAddLayout = async (extension: ExtensionDocument) => {
 		await modules.storage.put(extension);
 		await router.navigate(extension._id);
 	};
```

Comment:
hey @dev-tronghieu, could you help me convert these code into a reusable method, please? I could use it again in `walless-adapter`